common-npm-packages/codeanalysis-common/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "azure-pipelines-tasks-codeanalysis-common",
-  "version": "2.0.1",
+  "version": "2.0.2",
```

Comment:
Should we update `package-lock.json` also?

Reply:
@max-zaytsev Yes, I updated it.